Components/Basic/custom.component.yaml.meta.yaml

```diff
@@ -1,7 +1,33 @@
 ---
 docstring: |
     ## custom.component.yaml
-    *TODO*
+    Use a custom Python component.
+    ### Properties
+    You may pass data into the component by defining custom variables in the `properties` field. Within the custom component these variables can be accessed using:
+    `self.properties.get("my_property_name")`
+    ### Example
+    Here is an example custom component. The flow state uses the component and passes two arguments to the component.
```

Comment:
Better language would be: In this example, 2 user-defined properties are passed to the custom component.